libraries/holochain-service-client/src/androidTest/androidTest.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this file need to be added to git?

Reply:
Looks like it is an IDE artifact from android studio. Removed https://github.com/holochain/android-service-runtime/pull/93/commits/b11642f6d8bb9d6fdf4da3a0a42773931c8b0457